logs/logs.go

```diff
@@ -19,10 +19,10 @@ type Cmd struct {
 }
 
 type logsCmd struct {
-	Follow bool          `help:"Follow the logs by live tailing." short:"f"`
-	Lines  int           `help:"Amount of lines to output" default:"10" short:"l"`
-	Since  time.Duration `help:"Duration how long to look back for logs" short:"s" default:"10m"`
-	Output string        `help:"Configures the log output format. ${enum}" short:"o" enum:"default,json" default:"default"`
+	Follow bool           `help:"Follow the logs by live tailing." short:"f"`
+	Lines  int            `help:"Amount of lines to output" default:"10" short:"l"`
+	Since  *time.Duration `help:"Duration how long to look back for logs" short:"s" default:"60m"`
```

Comment:
Even though this is a pointer, kong will still default to 60m. I think we have to use a different approach. We should use the `Since` value whenever the build name is not specified. If the build name is specified I don't think there's ever a reason to query logs before the build as there won't be anything there anyway.

Reply:
changed it back to a normal value 👍